neo/public/javascripts/timeseries/controllers.js

```diff
@@ -1,29 +1,57 @@
 angular.module('cloudberry.timeseries', ['cloudberry.common'])
-  .controller('TimeSeriesCtrl', function ($scope, $window, Asterix) {
+  .controller('TimeSeriesCtrl', function ($scope, $window, $compile, Asterix) {
     $scope.ndx = null;
     $scope.result = {};
     $scope.resultArray = [];
     $scope.d3 = $window.d3;
     $scope.dc = $window.dc;
     $scope.crossfilter = $window.crossfilter;
     $scope.empty = [];
+    $scope.currentTweetCount = 0;
+    $scope.formatNumber = function (number) {
+      return number.toLocaleString('en-US');
+    };
+    $scope.totalCountString = "0";
+    $scope.currentTweetCountString = "0";
     for (var date = new Date(); date >= Asterix.startDate; date.setDate(date.getDate()-1)) {
       $scope.empty.push({'time': new Date(date), 'count': 0});
     }
     $scope.preProcess = function (result) {
       // TODO make the pattern can be changed by the returned result parameters
       var result_array = [];
+      $scope.currentTweetCount = 0;
       if (result && result[0]) {
         var granu = Object.keys(result[0])[0];
         angular.forEach(result, function (value, key) {
           key = new Date(value[granu]);
           value = +value.count;
+          $scope.currentTweetCount += value;
           result_array.push({'time': key, 'count': value});
         });
+        $scope.currentTweetCountString = $scope.formatNumber($scope.currentTweetCount);
 
       }
       return result_array;
     };
+
+    // add information about the count of tweets
+    var countDiv = document.createElement("div");
+    countDiv.id = "countDiv";
+    countDiv.title = "Display the count information of Tweets";

```

Comment:
The value of the title attribute will be displayed as a tooltip when you mouse over the paragraph